docs/features/kubernetes/installation.md

```diff
@@ -175,8 +175,7 @@ export default async function createPlugin(
 
 ## Running Backstage locally
 
-Start the frontend and the backend app by
-[running Backstage locally](../../getting-started/running-backstage-locally.md).
+Start the frontend and the backend app by running `yarn dev`
```

Comment:
Reading this page, I'd wonder why the Kuberentes feature even HAS this section?  Surely if I want to run the app locally, I'd go to the getting started section. I wouldn't look for that instruction on each of the features page.

Reading through this section, I'd actually suggest to remove both this line and the heading for this entire section?

The installation subnav currently looks like this:

<img width="260" alt="image" src="https://user-images.githubusercontent.com/33203301/231849245-9eaf0404-4920-45e5-985d-a0f621626ac2.png">

And so I think the local section isn't valuable here.

Reply:
Works for me, it's gone now 👍